### ISSUE TITLE
Does thundersvm support multi-GPU training?will multi-GPU make the training faster?

### ISSUE THREAD

**Issue by zccyman** [zeyiwen/thundersvm#28]

**Comment by zeyiwen** [comment]
No, it doesn't support multi-GPUs. I think multi-GPUs can help improve efficiency, especially when performing cross-validation or multi-class SVMs.

**Comment by bingshenghe** [comment]
It will be an interesting problem to extend ThunderSVM to multiple GPUs. are you interested in working into this?

**Comment by ZesenChen** [comment]
> No, it doesn't support multi-GPUs. I think multi-GPUs can help improve efficiency, especially when performing cross-validation or multi-class SVMs.

Hello，zeyiwen. I have a question that if 0-th GPU has no memory left. Will thundersvm switch to another one automatically？

**Comment by zeyiwen** [comment]
Hi @ZesenChen 
It doesn't support this functionality, but you can specify the GPU id using ```-u``` option (cf. [parameter](https://thundersvm.readthedocs.io/en/latest/parameters.html)).
